flake.nix

```diff
@@ -116,7 +116,7 @@
         else pkgs.clangStdenv;
       rustFlagsEnv =
         if stdenv.isLinux
-        then ''$RUSTFLAGS -C link-arg=-fuse-ld=lld -C target-cpu=native -Clink-arg=-Wl,--no-rosegment''
+        then ''''${RUSTFLAGS:-""} -C link-arg=-fuse-ld=lld -C target-cpu=native -Clink-arg=-Wl,--no-rosegment''
         else "$RUSTFLAGS";
```

Comment:
The `else` branch here will also need the explicit defaulting. Instead we can change the line below for exporting the custom rust flags to

```sh
export RUSTFLAGS="''${RUSTFLAGS:-""} ${rustFlagsEnv}"
```

and remove the `$RUSTFLAGS` from this variable altogether

Reply:
That's a cleaner approach indeed. Updated.